docs/reference/images/index.md

```diff
@@ -48,3 +48,37 @@ All public images available in {{brand}} support the following image properties:
 * `os_version=${VERSION_ID}`: operating system version, such as `os_version=22.04`
 
 [Other properties](https://docs.openstack.org/glance/latest/admin/useful-image-properties.html) may also be set on individual images.
+For instance, the `image_original_user` property reflects the username of the default non-root user of the corresponding operating system -- and that piece of information may come in handy in various automation scenarios.
+So, to find out the username of the default non-root user in the `Ubuntu 22.04 Jammy Jellyfish x86_64` image, you can type the following:
+
+```console
+$ openstack image show "Ubuntu 22.04 Jammy Jellyfish x86_64" \
+    -c properties -f json | jq '.[].image_original_user'
+
+"ubuntu"
```

Comment:
The combination of `-c <field>` with `-f json` strikes me as odd. I'd suggest: 
```suggestion
$ openstack image show -f json "Ubuntu 22.04 Jammy Jellyfish x86_64" \
    | jq '.properties.image_original_user'
"ubuntu"
```

Reply:
Done!